superset-frontend/src/SqlLab/components/ExploreCtasResultsButton/index.jsx

```diff
@@ -35,29 +35,24 @@ const propTypes = {
   templateParams: PropTypes.string,
 };
 
-class ExploreCtasResultsButton extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.visualize = this.visualize.bind(this);
-    this.onClick = this.onClick.bind(this);
-  }
 
```

Comment:
There should only be one space here, frontend CI will fail on this, please remove :)